health/egov-persister/plan-service-persister.yml

```diff
@@ -494,3 +494,29 @@ serviceMaps:
             - jsonPath: $.Plan.auditDetails.lastModifiedBy
 
             - jsonPath: $.Plan.auditDetails.lastModifiedTime
+
+    - version: 1.0.0
+      description: Updates the plan facility
+      fromTopic: update-plan-facility
+      isTransaction: true
+      queryMaps:
+        - query: UPDATE plan_facility_linkage SET residing_boundary = ?, service_boundaries = ?, additional_details = ?, active = ?,  last_modified_by=?, last_modified_time=? WHERE id = ?;
+          basePath: $.PlanFacility
+          jsonMaps:
+            - jsonPath: $.PlanFacility.residingBoundary
+
+            - jsonPath: $.PlanFacility.serviceBoundaries
```

Comment:
why is serviceBoundaries of type JSONB, in the db design its of type TEXT

Reply:
Corrected to text